numpy/distutils/ccompiler_opt.py

```diff
@@ -2325,7 +2338,8 @@ def report(self, full=False):
             "unix-like"   if self.cc_is_nocc   else self.cc_name)
         ))
         ########## baseline ##########
-        if self.cc_noopt:
+        # TODO: Fix this
```

Comment:
CI was passing - what still needs doing here?

Reply:
Not needed, so removed.